src/components/Dashboard.vue

```diff
@@ -192,7 +194,9 @@ export default {
       }
 
       // Distribution of course (e.g. MQR-AS)
-      const distributions = course.distributions || course.catalogDistr.split(',');
+      // alternateDistributions option in case catalogDistr for the course is null
+      const alternateDistributions = (course.catalogDistr != null) ? course.catalogDistr.split(', ') : [""];
```

Comment:
Clean out parentheses

Reply:
Used a regex to clean out the parentheses!